menas/ui/components/Component.js

```diff
@@ -14,7 +14,7 @@
  */
 
 sap.ui.define([
-    "sap/ui/core/UIComponent"
+  "sap/ui/core/UIComponent",
```

Comment:
There's only one element in that array - why the comma?

Reply:
thnx, #208 Clean up according to the comments